packages/ui/src/components/va-option-list/VaOptionList.vue

```diff
@@ -123,7 +123,7 @@ export default defineComponent({
 
     const isDisabled = (option: SelectableOption) => props.disabled || getDisabled(option)
 
-    const reset = () => { valueComputed.value = undefined }
+    const reset = () => { valueComputed.value = null }
```

Comment:
probably should be `undefined`, `null` is more of domain logic thing.

Reply:
In the component props definition added support only for the given types:
`defaultValue: ({ type: [String, Number, Object, Array] as PropType<OptionListValue | null> })`
I thought I need to use `null` here.